Objects/typeobject.c

```diff
@@ -10877,6 +10862,63 @@ recurse_down_subclasses(PyTypeObject *type, PyObject *attr_name,
     return 0;
 }
 
+static int
+expect_manually_inherited(PyTypeObject *type, void **slot)
```

Comment:
This all seems a bit ad-hoc.
This is just for 3.13 though, and we will use something more robust for 3.14 onward?

Reply:
Correct.  For 3.14+ we do it differently.